packages/react-native-codegen/src/parsers/flow/modules/__tests__/__snapshots__/module-parser-snapshot-test.js.snap

```diff
@@ -5,12 +5,9 @@ exports[`RN Codegen Flow Parser Fails with error message EMPTY_ENUM_NATIVE_MODUL
 exports[`RN Codegen Flow Parser Fails with error message MAP_WITH_EXTRA_KEYS_NATIVE_MODULE 1`] = `"Module NativeSampleTurboModule: 'ObjectTypeAnnotation' cannot contain both an indexer and properties."`;
 
 exports[`RN Codegen Flow Parser Fails with error message MIXED_VALUES_ENUM_NATIVE_MODULE 1`] = `
-"Syntax error in path/NativeSampleTurboModule.js: cannot use string initializer in number enum (19:2)
-  STR = 'str',
-  ^~~~~~~~~~~
-note: start of enum body (17:21)
-export enum SomeEnum {
-                     ^"
+"Syntax error in path/NativeSampleTurboModule.js: 'true', 'false', 'string', 'number' or 'bigint' expected in enum member initializer (19:17)
```

Comment:
Yeah, see, we want to capture the failure case of the other one too. So this is why you'll need another fixture

Reply:
Fixed 👍🏻